### PR TITLE
[feature](nereids) using rollup column stats

### PR DESCRIPTION
**Pull request by englefly**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by englefly** [comment]
run buidall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java [hunk 1]
```diff
@@ -649,6 +649,11 @@ private Statistics computeCatalogRelation(CatalogRelation catalogRelation) {
         TableIf table = catalogRelation.getTable();
         double rowCount = catalogRelation.getTable().estimatedRowCount();
         boolean hasUnknownCol = false;
+        long idxId = -1;
+        if (catalogRelation instanceof OlapScan) {
+            OlapScan olapScan = (OlapScan) catalogRelation;
+            idxId = olapScan.getSelectedIndexId();
```
Comment (Jibing-Li) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java:655]: What if the selected index is base index? I think in this case, we need to set idxId back to -1. Something like:
```
if (idxId == ((OlapTable) table).getBaseIndexId()) {
idxId = -1
}
```

Comment (englefly) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java]: done

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run pipelinex_p0

**Review by morrySnow — APPROVED** [review]

**Review by Jibing-Li — APPROVED** [review]
LGTM

**Pull request merged**